sympy/solvers/solveset.py

```diff
@@ -2514,6 +2596,18 @@ def linear_eq_to_matrix(equations, *symbols):
     The only simplification performed is to convert
     ``Eq(a, b)`` $\Rightarrow a - b$.
 
+    To suppress the error raised when nonlinearity is detected, pass
+    keyword `strict=False` and use the result with caution. There will
+    be a naive separation of coefficients from literal occurances of
+    the indicated symbols with limited error reporting.
+
+    Flags
+    =====
```

Comment:
This should be called parameters per the style guide:
https://docs.sympy.org/latest/guides/contributing/docstring.html#sections